cpu/ebizzy.py

```diff
@@ -0,0 +1,77 @@
+import os
+import json
+import re
+
+from avocado import Test
+from avocado import main
+from avocado.utils import archive
+from avocado.utils import process
+from avocado.utils import build
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Ebizzy(Test):
+
+    '''
+    ebizzy is designed to generate a workload resembling common web application
+    server workloads. It is highly threaded, has a large in-memory working set,
+    and allocates and deallocates memory frequently.
+    '''
+
+    def setUp(self):
+        '''
+        Build ebizzy
+        Source:
+        http://liquidtelecom.dl.sourceforge.net/project/ebizzy/ebizzy/0.3
+        /ebizzy-0.3.tar.gz
+        '''
+        sm = SoftwareManager()
+        if not sm.check_installed("gcc") and not sm.install("gcc"):
+            self.error("Gcc is needed for the test to be run")
+
+        tarball = self.fetch_asset('http://liquidtelecom.dl.sourceforge.net'
+                                   '/project/ebizzy/ebizzy/0.3'
+                                   '/ebizzy-0.3.tar.gz')
+        data_dir = os.path.abspath(self.datadir)
+        archive.extract(tarball, self.srcdir)
+        cb_version = os.path.basename(tarball.split('.tar.')[0])
+        self.srcdir = os.path.join(self.srcdir, cb_version)
+
+        patch = self.params.get(
+            'patch', default='Fix-build-issues-with-ebizzy.patch')
+        os.chdir(self.srcdir)
+        p1 = 'patch -p0 < %s/%s' % (data_dir, patch)
```

Comment:
`patch` is failing here.
~~~...
Running 'patch -p0 < /home/apahim/src/avocado-misc-tests/cpu/ebizzy.py.data/Fix-build-issues-with-ebizzy.patch'
[stdout] File configure is read-only; trying to patch anyway
[stdout] patching file configure
[stdout] Hunk #1 FAILED at 31.
[stdout] 1 out of 1 hunk FAILED -- saving rejects to file configure.rej
...
~~~
Could you please double-check?